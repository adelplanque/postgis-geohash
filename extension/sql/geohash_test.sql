CREATE EXTENSION postgis;
CREATE EXTENSION postgis_geohash;

SELECT GeohashAsInt64(ST_GeomFromText('POINT(10 20)'), 7);
SELECT GeohashAsText(ST_GeomFromText('POINT(10 20)'), 7);
SELECT GeohashAsInt64(ST_GeomFromText('LINESTRING(10 20, 30 40)'), 7);
SELECT GeohashAsText(ST_GeomFromText('LINESTRING(10 20, 30 40)'), 7);
