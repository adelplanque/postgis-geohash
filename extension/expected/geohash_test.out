CREATE EXTENSION postgis;
CREATE EXTENSION postgis_geohash;
SELECT GeohashAsInt64(ST_GeomFromText('POINT(10 20)'), 7);
   geohashasint64   
--------------------
 864691128455135232
(1 row)

SELECT GeohashAsText(ST_GeomFromText('POINT(10 20)'), 7);
 geohashastext 
---------------
 s0
(1 row)

SELECT GeohashAsInt64(ST_GeomFromText('LINESTRING(10 20, 30 40)'), 7);
 geohashasint64 
----------------
               
(1 row)

SELECT GeohashAsText(ST_GeomFromText('LINESTRING(10 20, 30 40)'), 7);
 geohashastext 
---------------
 
(1 row)

